File: src/debug/entry/javascript/problems/278.js
function runUserScript(userFunm, params, paramTypes) {
    if (params.length !== 2) {
        onParameterError();
    }
    const version = params[1];
    const n = params[0];
    const isBadVersion = function (k) {
        if (k >= version) {
            return true;
        }
        return false;
    }
    userFunm(isBadVersion)(n);
}


function parseSpecialParameter(index, type, param) {
    return null;
}

File: src/debug/entry/python3/problems/278.py
def runUserScript(func, params, paramTypes):
    if (len(params) != 2):
        onParameterError()

    version = params[1]
    n = params[0]

    def isBadVersion(k):
        if k > version:
            return True
        return False

    newParams = [n]
    func(*newParams)


def parseSpecialParameter(index, paramType, param):
    return None
